angularjs-portal-home/src/main/webapp/my-app/layout/list/partials/home-list-view.html

```diff
@@ -1,10 +1,13 @@
 <div class="row portlet-frame">
   <home-header></home-header>
   <div role="main" class="col-xs-12 no-padding portlet-body" ng-controller="LayoutController as layoutCtrl">
+    <div class='row col-xs-12' ng-show='showHomeFilter' style='margin: 5px; margin-left: 0;'>
+      <input ng-model="searchText" placeholder='Filter' class='form-control'>
```

Comment:
Maybe call this something other than `search`. `filterText` maybe?  Search makes it too easy to confuse with the top :mag: search bar :confused: .